packages/engine/Source/Scene/ArcGisMapServerImageryProvider.js

```diff
@@ -212,8 +220,10 @@ function ArcGisMapServerImageryProvider(options) {
   if (typeof credit === "string") {
     credit = new Credit(credit);
   }
-  this._credit = credit;
-
+  if (defined(options.warning)) {
```

Comment:
With this change, `this._credit` is no longer assigned unless `options.warning` is defined.This breaks the existing behavior– Now when not using `options.warning`, the provided `options.credit` will not show up at all.

I would suggest removing the `warning` parameter option from the constructor here, and instead in `fromBasemapType`, pass the `credit` option when creating the provider.

```
let credit;
if (defined(defaultTokenCredit)) {
    credit = Credit.clone(defaultTokenCredit);
}
return new ArcGisMapServerImageryProvider({
    ...options,
    url: server,
    token: accessToken,
    credit: credit
  });
```

Additionally, there should be no need to set `this._useTiles` in this case. It defaults to `true`, but you can also set `options.usePreCachedTilesIfAvailable` to `true` if you'd like to be explicit.

```
return new ArcGisMapServerImageryProvider({
    ...options,
    url: server,
    token: accessToken,
    credit: credit,
    usePreCachedTilesIfAvailable: true // ArcGIS Base Map Service Layers only support Tiled views
  });
```